### DISCUSSION TITLE
Error installing EnMap3.14 on QGIS 3.34

### DISCUSSION THREAD

**Discussion by omkamps** [EnMAP-Box/enmap-box#889] [Q&A]
I developed a remote sensing assignment for students using the EnMap-Box toolbox. Some, I recommended to update EnMap-Box because it resolved my problems with the classifier. However, they get the following error when installing the new QGIS and updating the EnMap-Box. 

Is there something we need to change because the error does not occur on my own computer. 

![IMG_20240507_112529](https://github.com/EnMAP-Box/enmap-box/assets/131457093/dc4a86ea-956f-4333-936f-af6ca87bcb6a)

**Comment by janzandr** [comment]
1. Have you restarted QGIS after the update?
2. Please copy&paste the error message as text.

**Reply by omkamps** [reply]
Apparently the error remains.

Couldn't load plugin 'enmapboxplugin' due to an error when calling its classFactory() method 
ImportError: cannot import name 'DMD_CREATIONFIELDDATASUBTYPES' from 'osgeo.gdalconst' (/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/GDAL-3.3.2-py3.9-macosx-10.13.0-x86_64.egg/osgeo/gdalconst.py)
Traceback (most recent call last):
  File "/Applications/QGIS-LTR.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/__init__.py", line 51, in classFactory
    plugin = EnMAPBoxPlugin(iface)
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/enmapboxplugin.py", line 54, in __init__
    enmapbox.initAll()
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/__init__.py", line 358, in initAll
    registerSpectralLibraryIOs()
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/qgispluginsupport/qps/__init__.py", line 176, in registerSpectralLibraryIOs
    initSpectralLibraryIOs()
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/qgispluginsupport/qps/speclib/core/spectrallibraryio.py", line 1066, in initSpectralLibraryIOs
    from ..io.geopackage import GeoPackageSpectralLibraryIO
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
    _module_ = __builtin_import__(name, *args, **kwargs)
  File "/Applications/QGIS-LTR.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 892, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/qgispluginsupport/qps/speclib/io/geopackage.py", line 6, in 
    from osgeo.gdalconst import DMD_CREATIONFIELDDATASUBTYPES
ImportError: cannot import name 'DMD_CREATIONFIELDDATASUBTYPES' from 'osgeo.gdalconst' (/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/GDAL-3.3.2-py3.9-macosx-10.13.0-x86_64.egg/osgeo/gdalconst.py)

Python version: 3.9.5 (default, Sep 10 2021, 16:18:19) [Clang 12.0.5 (clang-1205.0.22.11)] 
QGIS version: 3.34.6-Prizren Prizren, 623828f58c2 
Python Path:
•	/Applications/QGIS-LTR.app/Contents/MacOS/../Resources/python
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins
•	/Applications/QGIS-LTR.app/Contents/MacOS/../Resources/python/plugins
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/Rtree-0.9.7-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/cftime-1.2.1-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/geopandas-0.8.1-py3.9.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/statsmodels-0.11.1-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/scipy-1.5.1-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python39.zip
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/matplotlib-3.3.0-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/GDAL-3.3.2-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/netCDF4-1.5.4-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/lib-dynload
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/rasterio-1.1.5-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/numpy-1.20.1-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/patsy-0.5.1-py3.9.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/opencv_contrib_python-4.3.0.36-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/pandas-1.3.3-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/Fiona-1.8.13.post1-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/Pillow-7.2.0-py3.9-macosx-10.13.0-x86_64.egg
•	/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.9/site-packages/pyproj-3.2.0-py3.9-macosx-10.13.0-x86_64.egg
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/extlibs_darwin
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/qgispluginsupport/qps/pyqtgraph
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/apps
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/coreapps
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/eo4qapps
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/site-packages
•	/Users/sannevanoudenaarde/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/site-packages

**Comment by janzandr** [comment]
That is a known issue. Please look here:
https://github.com/EnMAP-Box/enmap-box/issues/886
